### PR TITLE
ajout de la page coworking

### PR DESCRIPTION
**Pull request by lunika**

### CODE REVIEW

File: images/cowork.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0,0-1.41-1.41l-3.42,3.42a1,1,0,1,0,1.41,1.41Z"/><path d="M69.29,40.56l-5.87,5.88a1,1,0,1,0,1.41,1.41L70.71,42a1,1,0,0,0-1.41-1.41Z"/><path d="M89,10H46a1,1,0,0,0-1,1V31a1,1,0,0,0,1,1H89a1,1,0,0,0,1-1V11A1,1,0,0,0,89,10ZM88,30H47V12H88Z"/><path d="M50,29h9a1,1,0,0,0,1-1V14a1,1,0,0,0-1-1H50a1,1,0,0,0-1,1V28A1,1,0,0,0,50,29Zm1-14h7V27H51Z"/><path d="M65,20h7a1,1,0,0,0,1-1V14a1,1,0,0,0-1-1H65a1,1,0,0,0-1,1v5A1,1,0,0,0,65,20Zm1-5h5v3H66Z"/><path d="M77,20h7a1,1,0,0,0,1-1V14a1,1,0,0,0-1-1H77a1,1,0,0,0-1,1v5A1,1,0,0,0,77,20Zm1-5h5v3H78Z"/><path d="M65,29h7a1,1,0,0,0,1-1V23a1,1,0,0,0-1-1H65a1,1,0,0,0-1,1v5A1,1,0,0,0,65,29Zm1-5h5v3H66Z"/><path d="M77,29h7a1,1,0,0,0,1-1V23a1,1,0,0,0-1-1H77a1,1,0,0,0-1,1v5A1,1,0,0,0,77,29Zm1-5h5v3H78Z"/><path d="M56,17H53a1,1,0,0,0,0,2h3a1,1,0,0,0,0-2Z"/><path d="M56,20H53a1,1,0,0,0,0,2h3a1,1,0,0,0,0-2Z"/><text x="0" y="115" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">Created by Maria Kislitsina</text><text x="0" y="120" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">from the Noun Project</text></svg>
```
Comment (jmaupetit) [inline images/cowork.svg]: Do you have a worst image to integrate please? :trollface:

**Comment by krichprollsch** [comment]
The menu is too long now :(
capture d ecran de 2018-07-19 16-20-30

**Comment by krichprollsch** [comment]
We could maybe group dejdev with C&C in a common page?

**Comment by jmaupetit** [comment]
> We could maybe group dejdev with C&C in a common page?

Or archive DejDev that no longer exist.

File: images/cowork.svg (hunk 1 above)
Comment (lunika) [reply images/cowork.svg]: yes

File: coworking.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+[Le bivouac](http://www.lebivouac.com/) propose une offre de coworking pour tous les développeurs qui participent aux évènements Clermont'ech. Les locaux du bivouac sont situés dans [l'Hôtel de région](https://osm.org/go/0AkOKx_14--).
+
+Pour profiter de cette offre dédiée aux développeurs, vous devez nous contacter par mail : <a href="mailto:cowork@clermontech.org">cowork@clermontech.org</a>
+
+Les tarifs négociés sont les suivants : 
+
+- 6€ HT la demi-journée.
+- 66€ HT à mi-temps.
+- 117€ HT à temps complet.
```
[the remark this answers is not in the document]
Comment (lunika) [reply coworking.md]: I think yes. It's boring when you are looking for a coworking place and you don't find the price.

**Comment by lunika** [comment]
PR updated @krichprollsch

**Pull request merged**